src/utils/getCallbackRoute.ts

```diff
@@ -0,0 +1,22 @@
+import { getIsProviderEqualTo } from 'utils/sdkDappUtils';
+import { LoginMethodsEnum } from 'types/sdkDappTypes';
+
+type GetCallbackRouteProps = {
+  anchor?: string;
+};
+
+export const getCallbackRoute = ({ anchor }: GetCallbackRouteProps) => {
+  const isWebWallet = getIsProviderEqualTo(LoginMethodsEnum.wallet);
```

Comment:
try not to access the state in a stateles function. Maybe create a hook and get the provider from the store